README.md

```diff
@@ -1,78 +1,109 @@
+
+## Action Outputs
+
+### Pull Request Comment
+This is shown on the pull request when the `create-pr-comment` is set to `true` and there is a PR associated with the commit.
+<kbd><img src="./docs/pull_request_comment.png"></img></kbd>
+
+### Pull Request Status Check
+This is shown on the pull request when the `create-status-check` is set to `true` and there is a PR associated with the commit.
+<kbd><img src="./docs/pull_request_status_check.png"></img></kbd>
+
+### Workflow Run
+This is shown on the workflow run when the `create-status-check` is set to `true`.
+<kbd><img src="./docs/status_check.png"></img></kbd>
+
+### Code Coverage Details
+If the `Code Coverage Details` in the Status Check body or PR Comment are expanded a summary similar to this is shown:
+<kbd><img src="./docs/code_coverage_details.png"></img></kbd>
+
 
 ## Inputs
-| Parameter | Is Required | Default | Description           |
-| --------- | ----------- | ------- | --------------------- |
-| `input-1` | true        |         | Description goes here |
-| `input-2` | false       |         | Description goes here |
+| Parameter                   | Is Required | Default               | Description                                                                                                                                                         |
+| --------------------------- | ----------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `github-token`              | true        | N/A                   | The GitHub token for interacting with the repository.                                                                                                               |
+| `summary-file`              | true        | N/A                   | The summary file generated by the report-generator action.                                                                                                          |
+| `report-name`               | false       | Code Coverage Results | The desired name of the report that is shown on the PR Comment and inside the Status Check.                                                                         |
+| `check-name`                | false       | code coverage         | The desired name of the status check.                                                                                                                               |
+| `create-status-check`       | false       | true                  | Flag indicating whether a status check with code coverage results should be generated.                                                                              |
+| `create-pr-comment`         | false       | true                  | Flag indicating whether a PR comment with code coverage results should be generated.                                                                                |
```

Comment:
I noticed a few of these are marked required in the action.yml file. Are you listing them as not required here because they have a default value?

Reply:
Yeah since this is more of what the user needs to provide.

Reply:
Sounds good to me!